deliverystream/src/main/java/com/amazonaws/kinesisfirehose/deliverystream/HandlerUtils.java

```diff
@@ -249,6 +269,51 @@
 				.build();
 	}
 
+    static software.amazon.awssdk.services.firehose.model.HttpEndpointDestinationConfiguration translateHttpEndpointDestinationConfiguration(
+            final HttpEndpointDestinationConfiguration httpEndpointDestinationConfiguration) {
+        if(httpEndpointDestinationConfiguration == null) return null;
+        return software.amazon.awssdk.services.firehose.model.HttpEndpointDestinationConfiguration.builder()
+                .bufferingHints(translateHttpEndpointBufferingHints(httpEndpointDestinationConfiguration.getBufferingHints()))
+                .requestConfiguration(translateHttpEndpointRequestConfiguration(httpEndpointDestinationConfiguration.getRequestConfiguration()))
+                .endpointConfiguration(translateHttpEndpointConfiguration(httpEndpointDestinationConfiguration.getEndpointConfiguration()))
+                .cloudWatchLoggingOptions(translateCloudWatchLoggingOptions(httpEndpointDestinationConfiguration.getCloudWatchLoggingOptions()))
+                .processingConfiguration(translateProcessingConfiguration(httpEndpointDestinationConfiguration.getProcessingConfiguration()))
+                .retryOptions(translateHttpEndpointRetryOptions(httpEndpointDestinationConfiguration.getRetryOptions()))
+                .roleARN(httpEndpointDestinationConfiguration.getRoleARN())
+                .s3BackupMode(httpEndpointDestinationConfiguration.getS3BackupMode())
+                .s3Configuration(translateS3DestinationConfiguration(httpEndpointDestinationConfiguration.getS3Configuration()))
+                .build();
+    }
+
+    static software.amazon.awssdk.services.firehose.model.HttpEndpointConfiguration translateHttpEndpointConfiguration(
+            final com.amazonaws.kinesisfirehose.deliverystream.HttpEndpointConfiguration httpEndpointConfiguration) {
+        if(httpEndpointConfiguration == null) return null;
+        return software.amazon.awssdk.services.firehose.model.HttpEndpointConfiguration.builder()
+                .accessKey(httpEndpointConfiguration.getAccessKey())
+                .name(httpEndpointConfiguration.getName())
+                .url(httpEndpointConfiguration.getUrl())
+                .build();
+    }
+
+    static software.amazon.awssdk.services.firehose.model.HttpEndpointRequestConfiguration translateHttpEndpointRequestConfiguration(
+            final com.amazonaws.kinesisfirehose.deliverystream.HttpEndpointRequestConfiguration httpEndpointDestinationConfiguration) {
```

Comment:
Can probably rename to `httpEndpointRequestConfiguration` instead of `httpEndpointDestinationConfiguration`?